scss/style.scss

```diff
@@ -0,0 +1,1292 @@
+                letter-spacing: 0.05em;
+                color: #212121;
+                margin: 4px 10px;
+            }
+        }
+
+        &__information {
+            font-family: OpenSans;
+            font-size: 15px;
+            line-height: 22px;
+            letter-spacing: 0.05em;
+            color: #212121;
+        }
+    }
+}
+
+
+.footer {
+    display: flex;
+    flex-direction: column;
+    background-color: #212121;
+    width: 100%;
+    height: 1254px;
+
+    &__contacts {
+        display: flex;
+        justify-content: center;
+
+    }
+
+    &__contact {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        margin-bottom: 84px;
+        width: 275px;
+        height: 202px;
+        margin-right: 25px;
+
+        &:last-child {
+            margin: 0px;
+        }
+
+        &__img {
+            width: 44px;
+            height: 62px;
+            background-color: #3949ab;
+            padding: 20px 32px;
+            border-radius: 50%;
+            margin-bottom: 40px;
+        }
+
+        &__text {
+            font-family: OpenSans;
+            font-size: 20px;
+            line-height: 28px;
+            letter-spacing: 0.05em;
+            color: #ffffff;
+            text-align: center;
+        }
+    }
+
+    &__data {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        width: 100%;
+    }
+
+    &__little-fields {
+        display: flex;
+        justify-content: center;
+        margin-bottom: 33px;
+    }
+
+    &__field {
+        width: 430px;
+        height: 84px;
+        border-radius: 10px;
+        font-family: OpenSans;
+        font-size: 20px;
+        letter-spacing: 0.05em;
+        color: #212121;
+        outline: none;
+        padding-left: 20px;
+    }
+
+    &__big-field {
+        width: 882px;
+        height: 300px;
+        border-radius: 10px;
+        font-family: OpenSans;
+        font-size: 20px;
+        letter-spacing: 0.05em;
+        color: #212121;
+        padding-left: 20px;
+        margin-bottom: 39px;
+        padding-top: 30px;
+    }
+
+    &__button {
+        display: flex;
+        justify-content: center;
+        width: 100%;       
+    }
+
+    &__button a {
+        text-decoration: none;
+        font-family: Roboto;
+        font-size: 20px;
+        font-weight: 700;
+        letter-spacing: 0.05em;
+        color: #212121;
+        background-color: #ffffff;
+        border-radius: 10px;
+        padding: 25px 82px;
+    }
+}
+
+.company {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+
+    &__text {
+        font-family: Roboto;
+        font-size: 20px;
+        line-height: 22px;
+        font-weight: 700;
+        letter-spacing: 0.05em;
+        color: #212121;
+    }
+}
+
+@media (min-width: 1020px) { 
+
+    .latest-works {
+
+        &__buttons--for-phones {
+            display: none;
+        }
+    }
+}
+
+// ТАБЛЕТКА
```

Comment:
Please use English language for all code/comments

Reply:
done